src/envoy/tcp/sni_verifier/sni_verifier_test.cc

```diff
@@ -136,7 +136,7 @@ INSTANTIATE_TEST_SUITE_P(
                     std::make_tuple(TLS1_2_VERSION, TLS1_2_VERSION),
                     std::make_tuple(TLS1_3_VERSION, TLS1_3_VERSION)));
 
-TEST_P(SniVerifierFilterTest, SnisMatch) {
+TEST_P(SniVerifierFilterTest, DISABLED_SnisMatch) {
```

Comment:
Is there an way we can keep track of those disabled tests and not forget about them?
Perhaps by adding a comment to all disabled tests in this file like `FIXME: Tests temporarily disabled, see: https://issues.redhat.com/projects/MAISTRA/issues/MAISTRA-XXXX` ? (Then we would need to create this issue and link it here)

Reply:
Done. Added a comment.

Reply:
Thanks!